spec/system/visiting_the_topics_view_page_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe 'visiting topics page' do 
+  scenario 'when viewing' do
+  	visit('/topics')
+  	expect(page).to have_content('Topics')
+  	expect(page).to have_content('What is your secret superpower?')
```

Comment:
I don't think this expectation can be passing, because you haven't specified any spec setup so your test database should be empty at this point (i.e. there won't be any topics to show). 

If this spec is passing then something is wrong with the way the specs are running (e.g. the specs might be running against your development database).